example/postgres/index.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const http = require('http');
+const { Client } = require('pg');
+const terminus = require('../../lib/terminus');
+const app = express();
+const client = new Client({
+  database: 'terminus',
+  user: 'test',
+  password: 'test'
+});
+
+app.get('/', async (req, res) => {
+  const result = await client.query('SELECT $1::text as message', ['Hello world!']);
+  res.send(result.rows[0].message);
+});
+
+function onHealthCheck() {
+  return Promise.resolve();
+}
+
+function onSignal() {
+  console.log('server is starting cleanup');
+  return client.end().then(() => console.log('client has disconnected'))
+    .catch(err => console.error('error during disconnection', err.stack));
+}
+
+process.on('exit', (code) => {
```

Comment:
what's the reason for this part? shouldn't we use the `onShutdown` option?

Reply:
Oh I forgot to remove this part `process.on('exit', (code) => {`, I was testing reactions on signals, because I'm working on Windows and default signal `SIGTERM` doesn't work (as I discovered in the [README.md](https://github.com/godaddy/terminus/blame/master/README.md#L84)), but `SIGINT` work. I will remove this part. 

I used `onSignal` option as comment in the [README.md](https://github.com/godaddy/terminus/blame/master/README.md#L18) suggest. I now that the `onShutdown` will work as expected, looking on the promise chain in the [terminus.js](https://github.com/godaddy/terminus/blame/master/lib/terminus.js#L52) file. If I change it to the `onShutdown` option this part in the [README.md](https://github.com/godaddy/terminus/blame/master/README.md#L18) file, should be change as well, I supposed. Should I change it to `onShutDown`?